tests/common.go

```diff
@@ -4397,6 +4399,66 @@ func collectAndCopyDiagsOnWorkerNodes(issueKey string) {
 	}
 }
 
+func collectAndCopyLogsFromPodsForTest(testCaseName string, podLabel map[string]string, namespace string, logLabel string) {
+	testCaseName = strings.ReplaceAll(testCaseName, " ", "")
+	podList, err := core.Instance().GetPods(namespace, podLabel)
+	if err != nil {
+		log.Errorf("Error in getting pods for the [%s] logs of test case [%s], Err: %v", logLabel, testCaseName, err.Error())
+		return
+	}
+	masterNode := node.GetMasterNodes()[0]
+	err = runCmd("pwd", masterNode)
+	if err != nil {
+		log.Errorf("Error in running [pwd] command in node [%s] for the [%s] logs of test case [%s]", masterNode.Name, logLabel, testCaseName)
+		return
+	}
+	testCaseLogDirPath := fmt.Sprintf("%s/%s-logs", pxbLogDirPath, testCaseName)
+	log.Infof("Creating a directory [%s] in node [%s] to store [%s] logs for the test case [%s]", testCaseLogDirPath, masterNode.Name, logLabel, testCaseName)
+	err = runCmd(fmt.Sprintf("mkdir -p %v", testCaseLogDirPath), masterNode)
+	if err != nil {
+		log.Errorf("Error in creating a directory [%s] in node [%s] to store [%s] logs for the test case [%s]. Err: %v", testCaseLogDirPath, masterNode.Name, logLabel, testCaseName, err.Error())
+		return
+	}
+	for _, pod := range podList.Items {
+		log.Infof("Writing [%s] pod into a %v/%v.log file", pod.Name, testCaseLogDirPath, pod.Name)
+		err = runCmd(fmt.Sprintf("kubectl logs %s -n %s > %s/%s.log", pod.Name, namespace, testCaseLogDirPath, pod.Name), masterNode)
+		if err != nil {
+			log.Errorf("Error in writing [%s] pod into a %v/%v.log file. Err: %v", pod.Name, testCaseLogDirPath, pod.Name, err.Error())
+		}
+	}
+}
+
+func collectAndCopyStorkLogsForPxBackupTest(testCaseName string) {
```

Comment:
Rename it is `collectStorkLogs`
Add comments

Reply:
Done.